src/cobra/core/dictlist.py

```diff
@@ -542,3 +543,28 @@ def __dir__(self) -> list:
         attributes.append("_dict")
         attributes.extend(self._dict.keys())
         return attributes
+
+    def to_df(self):
+        """Convert to a pandas dataframe."""
+        item = None
+        columns = []
+
+        if self:
```

Comment:
Maybe it's clearer to say

```suggestion
        if not self.empty():
```